### PR TITLE
add StarDist

### PR DESCRIPTION
**Pull request by FynnBe**
as mentioned in https://github.com/bioimage-io/collection-bioimage-io/issues/421 we want to an Application Card for StarDist. This PR adds such a BioImage.IO entry here, in our revived central bioimage-io-resources repository as stardist does not have its own partner repo.

@uschmidt83 @maweigert , please give us a 👍 to proceed or comment/contribute what needs changing.

### CODE REVIEW

**Comment by maweigert** [comment]
Thanks @FynnBe ! LGTM

File: collection.yaml [hunk 1]
```diff
@@ -35,4 +36,26 @@ config:  # todo: clean up config (which of these fields are still used?)
   default_type: model
   url_root: https://raw.githubusercontent.com/bioimage-io/bioimage-io-resources/main
   
-collection: []
+collection:
+- id: stardist
+  name: StarDist
+  description: "Object Detection with Star-convex Shapes"
+  format_version: 0.2.2
+  source: "https://github.com/stardist/stardist"
```
Comment (constantinpape) [inline collection.yaml]: Don't we need source to be the link to an actual bioengine app?

Comment (FynnBe) [reply collection.yaml]: afaik no. @oeway , correct me if I'm wrong. 
If we do have such requirement we should add it to the spec. I would expect the website to just not make a runnable BioEngine app from it, but just display it like any other general RDF instead. 
If that is not the case we could also just remove `type: application` here and have it as a 'general' resource...?

Comment (oeway) [reply collection.yaml]: It doesn't need to be an runnable bioengine app, you can define it as an regular application (you need type: application), I would skip the source field, set github_repo instead. And it's better to add other field  ssuch as documentation, authors, tags.

**Review by constantinpape — APPROVED** [review]
Ok, good to go from my side now.

**Review by uschmidt83 — APPROVED** [review]

**Pull request merged**